src/app/showcase/page-showcase/page-showcase.component.html

```diff
@@ -15,6 +15,9 @@
         <li>Spacing of content</li>
     </ul>
 
+    <h2>Life-cycle hooks</h2>
+    <p>Be aware that using page with kirby-router-outlet, causes the views are being stacked instead of replaced. This means that you can't trust onInit and onDestroy. Instead you can trust the page events (enter) or (leave) instead.</p>
```

Comment:
```suggestion
    <p>Be aware that using page with <code>kirby-router-outlet</code>, causes the views to be stacked instead of replaced. This means that you can't trust <code>onInit</code> and <code>onDestroy</code>. Instead you can trust the <code>kirby-page</code> events <code>(enter)</code> and <code>(leave)</code> instead.</p>
```